labelbox/data/annotation_types/label.py

```diff
@@ -138,6 +139,9 @@ def assign_feature_schema_ids(
 
         Note: You can now import annotations using names directly without having to lookup schema_ids
         """
+        warnings.warn("This method is deprecated and will be "
```

Comment:
Looks like there's one more `assign_feature_schema_ids` in this file under `LabelGenerator`

Reply:
Thanks for catching!